### PR TITLE
HADOOP-18387. Fix incorrect placeholder in hadoop-common

### PR DESCRIPTION
**Pull request by slfan1989**
JIRA: HADOOP-18387. Fix incorrect placeholder in hadoop-module.

### CODE REVIEW

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 49s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  1s |  |  codespell was not available.  |
| +0 :ok: |  detsecrets  |   0m  1s |  |  detect-secrets was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
|||| _ trunk Compile Tests _ |
| +0 :ok: |  mvndep  |  15m  8s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |  26m 21s |  |  trunk passed  |
| +1 :green_heart: |  compile  |  22m 45s |  |  trunk passed with JDK Private Build-11.0.15+10-Ubuntu-0ubuntu0.20.04.1  |
| +1 :green_heart: |  compile  |  20m  9s |  |  trunk passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  checkstyle  |   3m 51s |  |  trunk passed  |
| +1 :green_heart: |  mvnsite  |   3m 34s |  |  trunk passed  |
| +1 :green_heart: |  javadoc  |   2m 38s |  |  trunk passed with JDK Private Build-11.0.15+10-Ubuntu-0ubuntu0.20.04.1  |
| +1 :green_heart: |  javadoc  |   2m 13s |  |  trunk passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  spotbugs  |   4m 52s |  |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  20m 36s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 29s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   1m 59s |  |  the patch passed  |
| +1 :green_heart: |  compile  |  23m 48s |  |  the patch passed with JDK Private Build-11.0.15+10-Ubuntu-0ubuntu0.20.04.1  |
| +1 :green_heart: |  javac  |  23m 48s |  |  the patch passed  |
| +1 :green_heart: |  compile  |  21m 43s |  |  the patch passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  javac  |  21m 43s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  checkstyle  |   3m 38s |  |  the patch passed  |
| +1 :green_heart: |  mvnsite  |   3m 19s |  |  the patch passed  |
| +1 :green_heart: |  javadoc  |   2m 20s |  |  the patch passed with JDK Private Build-11.0.15+10-Ubuntu-0ubuntu0.20.04.1  |
| +1 :green_heart: |  javadoc  |   1m 54s |  |  the patch passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  spotbugs  |   5m  7s |  |  the patch passed  |
| +1 :green_heart: |  shadedclient  |  21m 47s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  unit  |  18m 32s |  |  hadoop-common in the patch passed.  |
| +1 :green_heart: |  unit  |   1m 17s |  |  hadoop-registry in the patch passed.  |
| +1 :green_heart: |  unit  |   1m  8s |  |  hadoop-yarn-applications-catalog-webapp in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 55s |  |  The patch does not generate ASF License warnings.  |
|  |   | 234m 56s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-4679/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/4679 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient spotbugs checkstyle codespell detsecrets |
| uname | Linux 50c104cd5ffe 4.15.0-169-generic #177-Ubuntu SMP Thu Feb 3 10:50:38 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 5899b2f5a67034420a2ad8cfc42390fe374cfe94 |
| Default Java | Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Private Build-11.0.15+10-Ubuntu-0ubuntu0.20.04.1 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07 |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-4679/1/testReport/ |
| Max. process+thread count | 3155 (vs. ulimit of 5500) |
| modules | C: hadoop-common-project/hadoop-common hadoop-common-project/hadoop-registry hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-catalog/hadoop-yarn-applications-catalog-webapp U: . |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-4679/1/console |
| versions | git=2.25.1 maven=3.6.3 spotbugs=4.2.2 |
| Powered by | Apache Yetus 0.14.0 https://yetus.apache.org |


This message was automatically generated.

File: hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/DelegationTokenRenewer.java [hunk 1]
```diff
@@ -256,9 +256,9 @@ public <T extends FileSystem & Renewable> void removeRenewAction(
       try {
         action.cancel();
       } catch (InterruptedException ie) {
-        LOG.error("Interrupted while canceling token for " + fs.getUri()
+        LOG.error("Interrupted while canceling token for {} filesystem." + fs.getUri()
```
Comment (steveloughran) [inline hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/DelegationTokenRenewer.java]: needs to move the fs.geturi to a param, not string concat

File: hadoop-common-project/hadoop-registry/src/main/java/org/apache/hadoop/registry/server/services/RegistryAdminService.java [hunk 2]
```diff
@@ -224,7 +224,7 @@ protected void serviceStart() throws Exception {
           bindingDiagnosticDetails(),
           dumpRegistryRobustly(true));
 
-      LOG.error(" Failure {}", e, e);
+      LOG.error(" Failure {}", e.getMessage(), e);
```
Comment (steveloughran) [inline hadoop-common-project/hadoop-registry/src/main/java/org/apache/hadoop/registry/server/services/RegistryAdminService.java]: -1. to this change

L227 is valid; it calls e.toString on demand and handles any errors raised. the second e forms the stack trace.

*e.getMessage() is null on some exceptions, like NPE*

**Review by steveloughran — CHANGES_REQUESTED** [review]
one line to change, one change to revert

File: hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/DelegationTokenRenewer.java (hunk 1 above)
Comment (slfan1989) [reply hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/DelegationTokenRenewer.java]: Thanks for the help reviewing the code, I will fix it.

File: hadoop-common-project/hadoop-registry/src/main/java/org/apache/hadoop/registry/server/services/RegistryAdminService.java (hunk 2 above)
Comment (slfan1989) [reply hadoop-common-project/hadoop-registry/src/main/java/org/apache/hadoop/registry/server/services/RegistryAdminService.java]: I will revert it.

**Comment by slfan1989** [comment]
@steveloughran Please help to review the code again, thank you very much!

File: hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/DelegationTokenRenewer.java [hunk 3]
```diff
@@ -256,9 +256,8 @@ public <T extends FileSystem & Renewable> void removeRenewAction(
       try {
         action.cancel();
       } catch (InterruptedException ie) {
-        LOG.error("Interrupted while canceling token for " + fs.getUri()
-            + "filesystem");
-        LOG.debug("Exception in removeRenewAction: {}", ie);
+        LOG.error("Interrupted while canceling token for {} filesystem." , fs.getUri());
```
Comment (ZanderXu) [inline hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/DelegationTokenRenewer.java]: Remove the useless space before the comma?

**Review by ZanderXu — CHANGES_REQUESTED** [review]

File: hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/DelegationTokenRenewer.java (hunk 3 above)
Comment (slfan1989) [reply hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/DelegationTokenRenewer.java]: @ZanderXu Thanks for the review, it has been fixed.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 49s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  1s |  |  codespell was not available.  |
| +0 :ok: |  detsecrets  |   0m  1s |  |  detect-secrets was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
|||| _ trunk Compile Tests _ |
| +0 :ok: |  mvndep  |  15m 18s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |  29m 27s |  |  trunk passed  |
| +1 :green_heart: |  compile  |  29m 37s |  |  trunk passed with JDK Private Build-11.0.15+10-Ubuntu-0ubuntu0.20.04.1  |
| +1 :green_heart: |  compile  |  25m 51s |  |  trunk passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  checkstyle  |   4m 52s |  |  trunk passed  |
| +1 :green_heart: |  mvnsite  |   4m 20s |  |  trunk passed  |
| +1 :green_heart: |  javadoc  |   3m  8s |  |  trunk passed with JDK Private Build-11.0.15+10-Ubuntu-0ubuntu0.20.04.1  |
| +1 :green_heart: |  javadoc  |   2m 52s |  |  trunk passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  spotbugs  |   6m  0s |  |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  25m 21s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 30s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   2m 25s |  |  the patch passed  |
| +1 :green_heart: |  compile  |  29m  0s |  |  the patch passed with JDK Private Build-11.0.15+10-Ubuntu-0ubuntu0.20.04.1  |
| +1 :green_heart: |  javac  |  29m  0s |  |  the patch passed  |
| +1 :green_heart: |  compile  |  25m 55s |  |  the patch passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  javac  |  25m 55s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| -0 :warning: |  checkstyle  |   4m 29s | [/results-checkstyle-root.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-4679/2/artifact/out/results-checkstyle-root.txt) |  root: The patch generated 1 new + 44 unchanged - 0 fixed = 45 total (was 44)  |
| +1 :green_heart: |  mvnsite  |   4m 18s |  |  the patch passed  |
| +1 :green_heart: |  javadoc  |   3m 12s |  |  the patch passed with JDK Private Build-11.0.15+10-Ubuntu-0ubuntu0.20.04.1  |
| +1 :green_heart: |  javadoc  |   2m 48s |  |  the patch passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  spotbugs  |   6m 34s |  |  the patch passed  |
| +1 :green_heart: |  shadedclient  |  24m 58s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  unit  |  19m 11s |  |  hadoop-common in the patch passed.  |
| +1 :green_heart: |  unit  |   1m 42s |  |  hadoop-registry in the patch passed.  |
| +1 :green_heart: |  unit  |   1m 33s |  |  hadoop-yarn-applications-catalog-webapp in the patch passed.  |
| +1 :green_heart: |  asflicense  |   1m 16s |  |  The patch does not generate ASF License warnings.  |
|  |   | 279m 53s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-4679/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/4679 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient spotbugs checkstyle codespell detsecrets |
| uname | Linux 059baa655d09 4.15.0-175-generic #184-Ubuntu SMP Thu Mar 24 17:48:36 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 820aaee3b4bdd9b5f9e65e65c229630950720fb5 |
| Default Java | Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Private Build-11.0.15+10-Ubuntu-0ubuntu0.20.04.1 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07 |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-4679/2/testReport/ |
| Max. process+thread count | 1235 (vs. ulimit of 5500) |
| modules | C: hadoop-common-project/hadoop-common hadoop-common-project/hadoop-registry hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-catalog/hadoop-yarn-applications-catalog-webapp U: . |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-4679/2/console |
| versions | git=2.25.1 maven=3.6.3 spotbugs=4.2.2 |
| Powered by | Apache Yetus 0.14.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 56s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  1s |  |  codespell was not available.  |
| +0 :ok: |  detsecrets  |   0m  1s |  |  detect-secrets was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
|||| _ trunk Compile Tests _ |
| +0 :ok: |  mvndep  |  15m  9s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |  30m  4s |  |  trunk passed  |
| +1 :green_heart: |  compile  |  30m 56s |  |  trunk passed with JDK Private Build-11.0.15+10-Ubuntu-0ubuntu0.20.04.1  |
| +1 :green_heart: |  compile  |  24m 20s |  |  trunk passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  checkstyle  |   4m 48s |  |  trunk passed  |
| +1 :green_heart: |  mvnsite  |   4m 26s |  |  trunk passed  |
| +1 :green_heart: |  javadoc  |   3m 27s |  |  trunk passed with JDK Private Build-11.0.15+10-Ubuntu-0ubuntu0.20.04.1  |
| +1 :green_heart: |  javadoc  |   2m 50s |  |  trunk passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  spotbugs  |   5m 39s |  |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  24m 44s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 33s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m  9s |  |  the patch passed  |
| +1 :green_heart: |  compile  |  30m  2s |  |  the patch passed with JDK Private Build-11.0.15+10-Ubuntu-0ubuntu0.20.04.1  |
| +1 :green_heart: |  javac  |  30m  2s |  |  the patch passed  |
| +1 :green_heart: |  compile  |  24m 30s |  |  the patch passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  javac  |  24m 30s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  checkstyle  |   4m 40s |  |  the patch passed  |
| +1 :green_heart: |  mvnsite  |   4m 22s |  |  the patch passed  |
| +1 :green_heart: |  javadoc  |   3m 20s |  |  the patch passed with JDK Private Build-11.0.15+10-Ubuntu-0ubuntu0.20.04.1  |
| +1 :green_heart: |  javadoc  |   2m 54s |  |  the patch passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  spotbugs  |   6m  0s |  |  the patch passed  |
| +1 :green_heart: |  shadedclient  |  25m  3s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  unit  |  19m  6s |  |  hadoop-common in the patch passed.  |
| +1 :green_heart: |  unit  |   1m 41s |  |  hadoop-registry in the patch passed.  |
| +1 :green_heart: |  unit  |   1m 28s |  |  hadoop-yarn-applications-catalog-webapp in the patch passed.  |
| +1 :green_heart: |  asflicense  |   1m 14s |  |  The patch does not generate ASF License warnings.  |
|  |   | 280m 27s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-4679/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/4679 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient spotbugs checkstyle codespell detsecrets |
| uname | Linux 14e76b9d5d7e 4.15.0-175-generic #184-Ubuntu SMP Thu Mar 24 17:48:36 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / dc5fad24c74e6d4b8d95c253613113161023c887 |
| Default Java | Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Private Build-11.0.15+10-Ubuntu-0ubuntu0.20.04.1 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07 |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-4679/3/testReport/ |
| Max. process+thread count | 1279 (vs. ulimit of 5500) |
| modules | C: hadoop-common-project/hadoop-common hadoop-common-project/hadoop-registry hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-catalog/hadoop-yarn-applications-catalog-webapp U: . |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-4679/3/console |
| versions | git=2.25.1 maven=3.6.3 spotbugs=4.2.2 |
| Powered by | Apache Yetus 0.14.0 https://yetus.apache.org |


This message was automatically generated.

**Review by steveloughran — APPROVED** [review]
LGTM. let's see what yetus says

**Review by ZanderXu — APPROVED** [review]
LGTM +1, Thanks @slfan1989

**Comment by slfan1989** [comment]
> LGTM. let's see what yetus says

@steveloughran Thanks for helping review the code!

**Comment by slfan1989** [comment]
> LGTM +1, Thanks @slfan1989

Thanks also for your help reviewing the code.

**Review by ayushtkn — APPROVED** [review]
LGTM

**Comment by slfan1989** [comment]
Thanks everyone!

**Pull request merged**